docs/community.aws.aws_acm_module.rst

```diff
@@ -5,7 +5,8 @@
 community.aws.aws_acm
 *********************
 
-**Upload and delete certificates in the AWS Certificate Manager service**
+**Upload and delete certificates in the AWS Certificate Manager service
+**
```

Comment:
```suggestion
**Upload and delete certificates in the AWS Certificate Manager service**
```

Reply:
Need to apply a change to aws_acm or this will just come back

```
diff --git a/plugins/modules/aws_acm.py b/plugins/modules/aws_acm.py
index d6ba255d..465d2cb0 100644
--- a/plugins/modules/aws_acm.py
+++ b/plugins/modules/aws_acm.py
@@ -28,7 +28,7 @@ __metaclass__ = type
 DOCUMENTATION = r'''
 ---
 module: aws_acm
-short_description: >
+short_description: >-
   Upload and delete certificates in the AWS Certificate Manager service
 version_added: 1.0.0
 description:
```

Reply:
I think I saw the fix aleady in the main branch,

Reply:
yes: https://github.com/ansible-collections/community.aws/commit/42688f3f1101c404e83dfb7f5be1280cfdec432f#diff-21a47213cb39a9205a965e7453b13c6b830fa0ceb2f2a69a789da50ec4e4f0f9R31 @tremble  
So this single change must be backport for future releases of stable-3

Reply:
ah yes, that's what your git diff is tolding.

Reply:
https://github.com/ansible-collections/community.aws/pull/1252 will backport this